src/liboslexec/wide/wide_opdictionary.cpp

```diff
@@ -191,7 +191,7 @@ template<typename ValueT> struct DictValueGetter<ValueT, true> {
             ustring attribname = wAttribName[lane];
             ElementType value[dest_array.length()];
             int result = context->dict_value(nodeID, attribname, wdest.type(),
-                                             &value[0]);
+                                             &value[0], false);
```

Comment:
Add comment on `false `argument identifying the name of parameter `/* treat_ustrings_as_hash */`